File: GymApp_v2/Data/MembershipType.cs
namespace GymApp_v1.Data
{
    /// Uygulamada kullanılacak üyelik türlerini tanımlar.
    public enum MembershipType
    {   
        None    = 0,  // “Seçilmedi” ya da başlangıç değeri
        Pool = 1,    // Havuz üyeliği
        Fitness = 2, // Fitness üyeliği
        Gold = 3     // Gold üyelik
    }
}

File: WebProjeOdevi/src/main/java/com/proje/fitnesapp/model/MembershipType.java
package com.proje.fitnesapp.model;

public enum MembershipType {
    POOL_ONLY("Havuz Üyeliği"),
    FITNESS_ONLY("Fitness Üyeliği"),
    BOTH("Gold Üyelik");

    private final String displayName;

    MembershipType(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}
